source/about/cloud-dedicated.rst

```diff
@@ -0,0 +1,120 @@
+Cloud Dedicated
+===============
+
+.. include:: ../_static/badges/ent-cloud-dedicated.rst
+  :start-after: :nosearch:
+
+Mattermost Cloud Dedicated is designed for larger organizations with higher demands for performance, scalability, customizability, and compliance looking to offload operational overhead and focus on more business-critical tasks.
+
+Your own private Mattermost instance running :ref:`Mattermost Enterprise <about/editions-and-offerings:mattermost enterprise>` is a Kubernetes cluster hosted and managed by Mattermost that runs on dedicated cloud infrastructure, where resources are exclusively available for your organization.
+
+Available features
+------------------
+
+Zero-downtime upgrades
+~~~~~~~~~~~~~~~~~~~~~~
+
+Mattermost releases biweekly updates and leverages recurring maintenance windows to keep your instance up-to-date with new stable or beta features behind feature flags, fix security issues, and ensure the overall reliability and performance of your environment. Maintenance windows are announced in advance on https://status.mattermost.com/
+
+Additional support options, including quicker response times, dedicated support personnel, and stronger service level agreements (SLAs), are also available.
+
+Disaster Recovery
+~~~~~~~~~~~~~~~~~
+
+Mattermost Cloud Dedicated supports data failover to a secondary region/site should the primary instance experiences an unrecoverable outage with guaranteed recovery times.
+
+Mattermost supports a **multi-AZ (availability zones)** strategy in the same site/region.
+
+Daily backups of the database, object storage, and high availability clusters are captured and retained for 30 days.
+
+In addition, highly available observability tools with automated alerting, long term metrics, and logs retention are retained for a duration of 1 year.
```

Comment:
For dedicated it will be good to add that we are able to maintain custom retention. @stylianosrigas thoughts?

Reply:
Yeap we can keep data for longer if the customer needs it ;)